astro/src/content/docs/_shared/_collected-metrics.mdx

```diff
@@ -0,0 +1,531 @@
+    "webhookJwtRefreshTokenRevokeTransactional": 7,
+    "webhookJwtRefreshTransactional": 6,
+    "webhookKickstartSuccess": 1,
+    "webhookUserAction": 4,
+    "webhookUserBulkCreate": 4,
+    "webhookUserBulkCreateTransactional": 7,
+    "webhookUserCreate": 4,
+    "webhookUserCreateComplete": 5,
+    "webhookUserCreateTransactional": 6,
+    "webhookUserDeactivate": 4,
+    "webhookUserDeactivateTransactional": 7,
+    "webhookUserDelete": 4,
+    "webhookUserDeleteComplete": 5,
+    "webhookUserDeleteTransactional": 6,
+    "webhookUserEmailUpdate": 4,
+    "webhookUserEmailVerified": 5,
+    "webhookUserEmailVerifiedTransactional": 6,
+    "webhookUserIdentityProviderLink": 5,
+    "webhookUserIdentityProviderUnlink": 4,
+    "webhookUserLoginIdDuplicateCreate": 5,
+    "webhookUserLoginIdDuplicateUpdate": 4,
+    "webhookUserLoginFailed": 5,
+    "webhookUserLoginFailedTransactional": 6,
+    "webhookUserLoginNewDevice": 5,
+    "webhookUserLoginNewDeviceTransactional": 6,
+    "webhookUserLoginSuccess": 4,
+    "webhookUserLoginSuccessTransactional": 7,
+    "webhookUserLoginSuspicious": 4,
+    "webhookUserLoginSuspiciousTransactional": 6,
+    "webhookUserPasswordBreach": 5,
+    "webhookUserPasswordBreachTransactional": 6,
+    "webhookUserPasswordResetSend": 4,
+    "webhookUserPasswordResetStart": 5,
+    "webhookUserPasswordResetSuccess": 4,
+    "webhookUserPasswordUpdate": 4,
+    "webhookUserReactivate": 5,
+    "webhookUserReactivateTransactional": 6,
+    "webhookUserRegistrationCreate": 4,
+    "webhookUserRegistrationCreateComplete": 4,
+    "webhookUserRegistrationCreateTransactional": 6,
+    "webhookUserRegistrationDelete": 4,
+    "webhookUserRegistrationDeleteComplete": 4,
+    "webhookUserRegistrationDeleteTransactional": 7,
+    "webhookUserRegistrationUpdate": 4,
+    "webhookUserRegistrationUpdateComplete": 4,
+    "webhookUserRegistrationUpdateTransactional": 6,
+    "webhookUserRegistrationVerified": 4,
+    "webhookUserRegistrationVerifiedTransactional": 6,
+    "webhookUserTwoFactorMethodAdd": 5,
+    "webhookUserTwoFactorMethodRemove": 4,
+    "webhookUserUpdate": 4,
+    "webhookUserUpdateComplete": 4,
+    "webhookUserUpdateTransactional": 6
+  }
+}
+```
+</details>
+
+You can see here that we are collecting information about feature usage.
+
+We will never collect anything personally identifiable about your users or customers. We may collect new metrics as we add new features, but those will be outlined in our release notes.
+
+Here's the breakdown of what FusionAuth deployments which this feature will collect data from, and how it will happen:
+
+-   Self-hosted: You get to choose whether or not you want to share your data with us. You can turn usage data collection off by API, setup wizard, or in the admin UI, under XXX. You can also block the outbound request at the network--just block the hostname https://usage-stats.fusionauth.io. If you decide to share usage data, we will protect it.
```

Comment:
XXX should be replaced with the path in the admin UI to turn off the stats gathering. Something like:

`<Breadcrumb>Settings -> Metrics</Breadcrumb>`

or whatever it is going to be.

Reply:
I'm torn about `If you decide to share usage data, we will protect it.`. I know I asked you to change it last time. I can't figure out what the right wording is to convey that we'll protect it the same way we do all our other data.

Reply:
Also, I'd change `Self-hosted: You get to choose` to `For those users who self-host FusionAuth, you can choose`

Reply:
Waiting for an answer on the first point. 

I think we drop that sentence. It reads a bit like "nice data...sure hope nothing happens to it."

Reply:
Makes sense to me to drop the 'protect your data' sentence.